docs/api/Apify.md

```diff
@@ -60,7 +60,7 @@ effect.
 </tr>
 <tr>
 <td colspan="3"><p>Array of event types, which you can set for actor run, see
-  the <a href="https://apify.com/docs/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.</p>
+  the <a href="https://docs.apify.com/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.</p>
```

Comment:
For example, this link is still wrong

`https://docs.apify.com/webhooks#events-actor-run`

should be

`https://docs.apify.com/webhooks/events#actor-run`

as the webhooks page was split into multiple once. Could you check out the links in this PR that contain hash part?